packages/ui/src/components/organisms/ProductDetails/styles.scss

```diff
@@ -178,6 +178,18 @@
     }
   }
 
+  [data-fs-product-details-description-content] {
```

Comment:
I trust @renatamottam or @hellofanny has reviewed or participated in this?

Reply:
yes, renata did.